tests/data_files/t_h_i_s__i_s__a__v_e_r_y__l_o_n_g__d_i_r_e_c_t_o_r_y__n_a_m_e_/t_o__e_n_s_u_r_e__t_h_a_t__m_b_e_d_t_l_s__d_o_e_s__t_h_e_/c_o_r_r_e_c_t__t_h_i_n_g__w_h_e_n__p_a_r_s_i_n_g_/a__v_e_r_y__l_o_n_g__p_a_t_h_/b_y__d_e_f_a_u_l_t__M_B_E_D_T_L_S___X_5_0_9___M_A_X___F_I_L_E___P_A_T_H___L_EN__i_s__s_e_t__to_2_5_6__b_y_t_e_s/t_h_i_s__i_s__a__v_e_r_y__l_o_n_g__d_i_r_e_c_t_o_r_y__n_a_m_e_/t_o__e_n_s_u_r_e__t_h_a_t__m_b_e_d_t_l_s__d_o_e_s__t_h_e_/c_o_r_r_e_c_t__t_h_i_n_g__w_h_e_n__p_a_r_s_i_n_g_/a__v_e_r_y__l_o_n_g__p_a_t_h_/b_y__d_e_f_a_u_l_t__M_B_E_D_T_L_S___X_5_0_9___M_A_X___F_I_L_E___P_A_T_H___L_EN__i_s__s_e_t__to_2_5_6__b_y_t_e_s/empty_test_fie.txt

```diff
@@ -0,0 +1,2 @@
+Empty test file used to test buffer overflow when parsing file path in
```

Comment:
To be able to test the functionality that this fixes we need a very long path (more than 512 bytes long). So I had to create a new file and multiple folders. If I do not create a deep hierarchy, we are at risk that the maximum filename is exceeded in OS.